src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -219,30 +226,29 @@ public SVCustomReadFilter(final UseEnd whichEnd,
                                   final boolean filterDupReads,
                                   final boolean filterSecondaryAlignments,
                                   final boolean filterSupplementaryAlignments,
-                                  final int MQThreshold){
+                                  final int MQThreshold,
+                                  final int mMQThreshold){
 
-                final UseEnd endVal = whichEnd;
+            final UseEnd endVal = whichEnd;
 
-                ReadFilter tempFilter = read -> 0!=read.getFragmentLength();
-                           tempFilter = tempFilter.and(read -> read.isPaired());
-                           tempFilter = tempFilter.and(read -> endVal == (read.isFirstOfPair() ? UseEnd.FIRST : UseEnd.SECOND));
+            ReadFilter tempFilter = read -> 0!=read.getFragmentLength();
+                       tempFilter = tempFilter.and(read -> read.isPaired());
+                       tempFilter = tempFilter.and(read -> endVal == (read.isFirstOfPair() ? UseEnd.FIRST : UseEnd.SECOND));
 
-                if(filterNonProperlyPairdReads)   { tempFilter = tempFilter.and(read -> read.isProperlyPaired());}
-                if(filterUnmappedReads)           { tempFilter = tempFilter.and(read -> !read.isUnmapped());}
-                if(filterMateUnmappedReads)       { tempFilter = tempFilter.and(read -> !read.mateIsUnmapped());}
-                if(filterDupReads)                { tempFilter = tempFilter.and(read -> !read.isDuplicate());}
-                if(filterSecondaryAlignments)     { tempFilter = tempFilter.and(read -> !read.isSecondaryAlignment());}
-                if(filterSupplementaryAlignments) { tempFilter = tempFilter.and(read -> !read.isSupplementaryAlignment());}
+            if(filterNonProperlyPairdReads)   { tempFilter = tempFilter.and(read -> read.isProperlyPaired());}
+            if(filterUnmappedReads)           { tempFilter = tempFilter.and(read -> !read.isUnmapped());}
```

Comment:
Why is this an option here? It wouldn't make sense to compute insert size on unmapped reads so I don't know why this would ever be false. Unless the goal is to have a filter that's reusable across tools, in which case I'd at least make this class be a stand-alone rather than an inner class in the Tool.